### PR TITLE
Slate says hi

### PR DESCRIPTION
**Pull request by kjhf**
- Moved get_presence_message to separate file
- Compatibility for no Google connector / local running without setup
- invalid_whatis choices -- remove empty response...
- Added Server cog and clear_checkin command

### CODE REVIEW

**Review by LeptoFlare — APPROVED** [review]
im late but looks good

WAIT IVE BEEN PUSHING TO YOUR BRANCH BY ACCIDENT THIS ENTIRE TIME

File: radia/cogs/info.py [hunk 1]
```diff
@@ -42,24 +48,54 @@ async def whatis(self, ctx, prefix=None, image: bool = False):
...
+                embed = utils.Embed(title="What Is...")
+                embed.add_field(
+                    name="Options:",
+                    value=utils.Embed.list(google.connector.whatis.options()))
+            except google.HollowSheet as e:
+                await ctx.send(e)
+            else:
+                await ctx.send(embed=embed)
 
     def invalid_whatis(self, prefix):
```
Comment (vlee489) [inline radia/cogs/info.py:69]: Please can we not have these random messages if an invalid `whatis` is asked for, and just say it does not exist please, these are frankly ridiculous.

Comment (vlee489) [reply radia/cogs/info.py:69]: if not ridiculous, some of them are rude

File: radia/__main__.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 # Create Bot
 intents = Intents.default()
 intents.members = True
-bot = Bot(command_prefix="!", intents=intents)
+bot = Bot(command_prefix="^", intents=intents)
```
Comment (vlee489) [inline radia/__main__.py:18]: This `bot = Bot(command_prefix="^", intents=intents)` does not look correct, should be 

```python
command_prefix="!" if not os.getenv("DEBUG") else "^"
```

**Review by vlee489 — CHANGES_REQUESTED** [review]
Mostly looks good, but a few changes are needed

File: radia/cogs/info.py (hunk 1 above)
Comment (vlee489) [reply radia/cogs/info.py:69]: If you want to keep these somewhere are an internal joke, put them into [Sheetbot](https://github.com/IPLSplatoon/SheetBot)

File: radia/__main__.py (hunk 2 above)
Comment (vlee489) [reply radia/__main__.py]: Fixed issue in Merge Conflict

File: radia/cogs/info.py (hunk 1 above)
Comment (LeptoFlare) [reply radia/cogs/info.py:74]: Resolved, see https://github.com/IPLSplatoon/Radia/pull/4/commits/0f5665aa2765644172a171b81938326530b1f673

The inside jokes only show up if the command is invoked inside of the IPL Staff server.
Additionally, I request that Radia be added to the IPL Staff server so you do not need to update sheetbot's code every time I add more inside jokes.

**Review by vlee489 — APPROVED** [review]

**Pull request merged**